src/woo_publications/api/tests/mixins.py

```diff
@@ -0,0 +1,181 @@
+from rest_framework import status
+from rest_framework.test import APIClient, APITestCase
+
+from .factories import TokenAuthFactory
+
+AUDIT_HEADERS = {
+    "AUDIT_USER_REPRESENTATION": "username",
+    "AUDIT_USER_ID": "id",
+    "AUDIT_REMARKS": "remark",
+}
+
+
+class APIKeyUnAuthorizedMixin:
+
+    self: APITestCase
+    client: APIClient
+
+    def assertWrongApiKeyProhibitsGetEndpointAccess(self, url):
+        no_permission_token = TokenAuthFactory.create().token
```

Comment:
```suggestion
        no_permission_token = TokenAuthFactory.create(permissions=[]).token
```
do not rely on implicit factory data for explicit test cases, set up your data explicitily to leave no doubt